custom_components/tahoma/manifest.json

```diff
@@ -1,8 +1,8 @@
 {
-  "domain": "tahoma",
+  "domain": "TaHoma",
```

Comment:
Don't change the domain and documentation URL, since this should be all lowercase.

Reply:
yep... I was to quick on the replace all. fixed already.